src/sql/workbench/contrib/query/browser/query.contribution.ts

```diff
@@ -404,6 +404,11 @@ const queryEditorConfiguration: IConfigurationNode = {
 			'description': localize('queryEditor.results.copyRemoveNewLine', "Configuration options for copying multi-line results from the Results View"),
 			'default': true
 		},
+		'queryEditor.results.avoidNewLineAfterTailingLineBreak': {
```

Comment:
What about this instead? I think that makes it a bit clearer that we normally add it, but don't in those situations. 

@erinstellato-ms Any thoughts?

(if you do decide to keep it - the name currently is "Tailing Line Break`. Not sure if you intended that vs. Trailing but Trailing is generally the term I've seen used)

```suggestion
		'queryEditor.results.skipNewLineAfterTrailingLineBreak': {
```

Reply:
oh that's a typo, had it correct in doc but not in variable name.... thanks for catching this let me fix it.